src/scores/categorical/contingency_impl.py

```diff
@@ -551,96 +554,153 @@ def true_negative_rate(self) -> xr.DataArray:
 
     def recall(self) -> xr.DataArray:
         """
-        Identical to probability of detection.
+        Identical to hit rate, probability of detection (POD), true positive rate and sensitivity.
 
         Calculates the proportion of the observed events that were correctly forecast.
 
         Returns:
-            An xarray object containing the probability of detection
+            xr.DataArray: An xarray object containing the probability of detection
 
         .. math::
             \\text{recall} = \\frac{\\text{true positives}}{\\text{true positives} + \\text{false negatives}}
 
         Notes:
             - Range: 0 to 1.  Perfect score: 1.
-            - "True positives" is the same as "hits"
-            - "False negatives" is the same as "misses"
+            - "True positives" is the same as "hits".
+            - "False negatives" is the same as "misses".
 
         References:
             - https://www.cawcr.gov.au/projects/verification/#POD
             - https://en.wikipedia.org/wiki/Precision_and_recall
         """
         return self.probability_of_detection()
 
-    def precision(self):
+    def precision(self) -> xr.DataArray:
         """
         Identical to the Success Ratio.
 
-        https://en.wikipedia.org/wiki/Precision_and_recall
+        What proportion of the forecast events actually eventuated?
+
+        Returns:
+            xr.DataArray: An xarray object containing the precision score
+
+        .. math::
+            \\text{precision} = \\frac{\\text{true positives}}{\\text{true positives} + \\text{false positives}}
+
+        Notes:
+            - Range: 0 to 1.  Perfect score: 1.
+            - "True positives" is the same as "hits"
+            - "False positives" is the same as "misses"
+
+        References:
+            - https://www.cawcr.gov.au/projects/verification/#SR
+            - https://en.wikipedia.org/wiki/Precision_and_recall
+
         """
         return self.success_ratio()
 
-    def f1_score(self):
+    def f1_score(self) -> xr.DataArray:
         """
         Calculates the F1 score.
-        https://en.wikipedia.org/wiki/F-score
+
+        Returns:
+            xr.DataArray: An xarray object containing the F1 score
+
+        .. math::
+            \\text{F1} = \\frac{\\text{true positives}}{\\text{true positives} + \\text{false positives} + \\text{false negatives}}
```

Comment:
You're missing two factors.

See the equation below in the implementation or the wikipedia page

Reply:
Yes, this looks wrong. I'll fix it. Thanks for picking it up.

Reply:
I've pushed up a fix to this mathjax, please take a look. I haven't updated 228 yet but I've fixed the source.